recipe/meta.yaml

```diff
@@ -38,129 +44,146 @@ build:
     - lib/R/etc/x64/ldpaths
     - lib/R/etc/x64/javaconf
   detect_binary_files_with_prefix: true
+  # The current openblas-devel run exports
+  # are overly restrictive IMHO, should be
+  # open-ended, but actually because we
+  # need to copy the shared library into this
+  # package and change its SONAME to libRblas.so
+  # we do not want a run dependency on openblas
+  # at all (otherwise Matrix.so ends up with a
+  # DT_NEEDED of libopenblas.so and cannot be used
+  # against any old libRblas.so)
+  ignore_run_exports:
+    - libopenblas
```

Comment:
Should this be `{{native}}openblas`

Reply:
No, the m2w64-* packages don't have run_exports in them. Therefore they need to be pinned.